plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -189,3 +193,22 @@ func defaultPriorities() sets.String {
 		),
 	)
 }
+
+func GetEquivalencePod(pod *api.Pod) interface{} {
+	equivalencePod := EquivalencePod{}
+	podSpec := &(pod.Spec)
+	for _, vol := range podSpec.Volumes {
+		equivalencePod.Volumes = append(equivalencePod.Volumes, vol)
+	}
+	equivalencePod.NodeSelector = podSpec.NodeSelector
+	equivalencePod.Request = predicates.GetResourceRequest(pod)
+	equivalencePod.Ports = predicates.GetUsedPorts(pod)
+	return &equivalencePod
+}
+
+type EquivalencePod struct {
+	Volumes      []api.Volume
+	NodeSelector map[string]string
+	Ports        map[int]bool
+	Request      *schedulercache.Resource
+}
```

Comment:
You are using EquivalencePod for both predicates and priorities purpose (please add a comment about it) [btw, I think we should move it a different place, but I'm not yet sure where]
Anyway, that means, that we are missing few things here - some that come to my mind are:
- namespace
- labels
- some annotations (e.g. PodAffinity)
@davidopp - am I right?
I'm wondering how to enforce that when someone will add a new predicate/prioritity this will be mirrorred here...